datalad/runner/tests/test_threadsafety.py

```diff
@@ -32,7 +33,7 @@ def pipe_data_received(self, fd, data):
             self.send_result((fd, line))
 
 
-def _runner_with_protocol(protocol) -> ThreadedRunner:
+def _runner_with_protocol(protocol: Type[WitlessProtocol]) -> ThreadedRunner:
```

Comment:
`from __future__ import annotations` lets you use the builtin `type` instead of `typing.Type` here.

Reply:
Thx. Changed it to `type[...]`